client/server-connection.cpp

```diff
@@ -170,9 +170,12 @@ namespace tec {
 			}
 		}
 
-		void ServerConnection::SyncHandler(const ServerMessage&) {
+		void ServerConnection::SyncHandler(const ServerMessage& message) {
 			std::chrono::milliseconds round_trip = std::chrono::duration_cast<std::chrono::milliseconds>(recv_time - sync_start);
 			std::lock_guard<std::mutex> recent_ping_lock(recent_ping_mutex);
+			if(message.GetBodyLength() >= 8) {
```

Comment:
What does 8 represent?

Speaking of the 10 constant just below this should be swapped for a constant.

Reply:
probably the low levelness getting the better of me, `sizeof(uint64_t)` would make more sense.